invenio_github/ext.py

```diff
@@ -27,6 +27,7 @@
 from __future__ import absolute_import, print_function
 
 from flask import current_app
+from invenio_stats.utils import obj_or_import_string
```

Comment:
Unfortunately, you'll have to copy the implementation from `invenio_stats/utils.py`  to `invenio_github/utils.py` and import it from there (otherwise, we'll have to add `invenio-stats` as a dependency of `invenio-github`). 

We were discussing to address this via e.g. introducing an `invenio-common` package, which has all these small Flask utilities, but that's a lot of changes across multiple modules...